python/ray/cloudpickle/cloudpickle_fast.py

```diff
@@ -114,9 +114,9 @@ def _class_getnewargs(obj):
     if "__slots__" in obj.__dict__:
         type_kwargs["__slots__"] = obj.__slots__
 
-    __dict__ = obj.__dict__.get('__dict__', None)
+    __dict__ = obj.__dict__.get("__dict__", None)
```

Comment:
There were a bunch of lint errors that I fixed here.

Reply:
Is it possible to skip linting the cloudpickle files? This would introduce difficulties when we sync with upstream cloudpickle later (the upstream cloudpickle often changes dramatically and they use different lint format, and it will be a mess when we try to adopt upstream changes). If possible, we would like to keep the difference between the upstream and our version minimal.

Reply:
Sounds good, I reverted these changes